CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ project adheres to [Semantic Versioning](http://semver.org/).
 ### Added
 
 - **Breaking:** Allow for specifying a custom AMI for the worker nodes. (by @bmcstdio)
+- Access conditional resource using join function in combination with splat syntax (by @miguelaferreira)
```

Comment:
@miguelaferreira This is a change for me and not a new feature. Can you move down this line to the **Changed** section.

Reply:
Sure, will do today